contracts/multisig-prover/src/execute.rs

```diff
@@ -241,22 +241,25 @@ pub fn update_worker_set(deps: DepsMut, env: Env) -> Result<Response, ContractEr
     }
 }
 
-pub fn confirm_worker_set(deps: DepsMut) -> Result<Response, ContractError> {
+pub fn confirm_worker_set(deps: DepsMut, info: MessageInfo) -> Result<Response, ContractError> {
```

Comment:
only the sender is necessary, so I would prefer just the sender as the function argument instead of `MessageInfo`

Reply:
Agree with @cgorenflo